source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,188 @@
+Shared Channels (Experimental) (E20)
+====================================
+
+Shared Channels bring people together from multiple Mattermost installations, such as teams collaborating with external partners, and customers using multiple Mattermost instances in a federated architecture. 
+
+Mattermost System Admins use slash commands to establish secure connections between Mattermost instances, then invite secured connections to shared channels.
+
+Shared channels behave like regular channels, offering the same user experience and functionality as Public and Private channels. All members of secure connections, including local members and remote members, can `send and receive messages <https://docs.mattermost.com/help/messaging/sending-messages.html#sending-and-receiving-messages>`__, `use emojis <https://docs.mattermost.com/help/messaging/emoji.html>`__ to react to messages, `share files <https://docs.mattermost.com/help/messaging/attaching-files.html>`__, and `search message history <https://docs.mattermost.com/help/getting-started/searching.html#searching-for-message-contents>`__. Content is synchronized in real-time across all participating Mattermost instances. 
+
+A channel’s permissions and access continues to be governed by each server separately. Channel moderation permissions can be applied to a shared channel, and be in effect on the local Mattermost server, while not being in effect on a remote Mattermost server.
+
+Setting Up Shared Channels
+--------------------------
+
+The process of sharing channels involves the following three steps:
+
+1. System Admins must enable Shared Channel functionality for their Mattermost instance. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#enable-shared-channels-experimental>`__ documentation for details.
+
+2. System Admins `use a slash command <https://docs.mattermost.com/help/messaging/executing-commands.html>`__ to establish a secure and trusted relationship between other Mattermost E20 instances. This process involves creating a password-protected, encrypted invitation, creating a strong decryption password, then sending the invitation and password to the System Admin of a remote Mattermost instance. We strongly recommend that you share the details of the secure connection invitation using a communication channel other than Mattermost, such as by email.
```

Comment:
Might be worth mentioning the best practise of communicating the invitation and password separately, meaning the invitation is sent independently relative to the password.  This ensures that if one is compromised the attacker won't have the other piece.